repose-aggregator/src/docs/asciidoc/filters/cors.adoc

```diff
@@ -1,20 +1,27 @@
-= CORS
-:toc:
-:icons: font
+= Cross-Origin Resource Sharing (CORS) Filter
 
 include::../_includes/in-progress.adoc[]
 
 The Cross-Origin Resource Sharing (CORS) filter allows Repose to manage CORS requests without the origin service needing to understand CORS.
-For an introduction to CORS, see the <<CORS Overview>> below.
+For an introduction to CORS, see the <<CORS Overview>> section below.
 
-WARNING: Before enabling the CORS filter, you should be familiar with the CORS specification and the associated security implications.
+[WARNING]
+====
+Before enabling the CORS filter, you should be familiar with the https://www.w3.org/TR/cors/[CORS specification] and the associated security implications.
 Enabling CORS can increase an endpoint's exposure.
-Configuring CORS improperly could expose CSRF vulnerabilities in the origin service that were previously blocked by not supporting CORS.
-
-== Headers
-
-=== Required
-The presence of certain headers will indicate what kind of request is being made.
+Configuring CORS improperly could expose https://en.wikipedia.org/wiki/Cross-site_request_forgery[Cross-Site Request Forgery (CSRF)] vulnerabilities in the origin service that were previously hidden by not supporting CORS.
+====
+
+== General filter information
+* *Name:* cors
+* *Default Configuration:* cors.cfg.xml
+* *Released:* v7.2.0.0
+* *Bundle:* repose-filter-bundle
+* *Schema:* link:../schemas/cors-configuration.xsd[cors-configuration.xsd]
```

Comment:
Yeah, my thinking was the "General Filter Information" would include the "Name" of the filter, and the section we're now calling "Examples" would be called "Configuration" with the "Default filename" and "Schema" listed there.  I'm guessing we decided against that based on what I saw in the Migrate Doc story which is why I went with this.  I'll make the changes you mentioned so we can feel it out.